webapp/channels/src/components/widgets/loading/loading_wrapper.tsx

```diff
@@ -6,24 +6,17 @@ import React from 'react';
 import LoadingSpinner from './loading_spinner';
 
 type Props = {
-    loading: boolean;
-    text: React.ReactNode;
-    children: React.ReactNode;
+    loading?: boolean;
```

Comment:
I don't think the optional arguments are necessary here if we're supporting default properties. `ReactNode` includes `null` in the union type and `booleans` are always truthy/falsy.

Reply:
Gotcha. I feel like this is something that could be improved with functional components, but I guess we make trade-offs for performance/other forms of simplicity at the expense of type correctness.